client/index.html

```diff
@@ -8,5 +8,6 @@
 	<body>
 		<div id="root"></div>
 		<script type="text/javascript" src="/bundle.js"></script>
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700' rel='stylesheet' type='text/css'>
```

Comment:
there seems to be some mixed tabs/spaces here. Perhaps move the whole file to spaces?